### PR TITLE
[MACAPA-2019] Adding Key Information

### PR DESCRIPTION
**Pull request by eujuniorweb**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: data/events/2019-macapa.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    url: /events/2019-macapa/contact
+  - name: codigo de conduta
+    url: /events/2019-macapa/conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "John Doe"
```
Comment (MikeRosTX) [inline data/events/2019-macapa.yml]: Please update this list with current organizers, and don't forget to send a list of all the organizer's **_first_name_**, **_last_name_**, and **_email_** to **_info@devopsdays.org_**.

Comment (eujuniorweb) [reply data/events/2019-macapa.yml]: Should I close this pull request, update the information, and create a new pull request?

Comment (MikeRosTX) [reply data/events/2019-macapa.yml]: No, just update the **_yml_** and sync it.

**Comment by MikeRosTX** [comment]
Please send an email with a list of all the organizer's **_first_name_**, **_last_name_**, and their **_email_** so that we can them to the team's email **_macapa@devopsdays.org_** and invite them to join Slack (Optional opt-in).

**Comment by MikeRosTX** [comment]
OK so I found all the organizer's emails. But these people are missing from this PR and we need them to be added before we can merge:

- Josseli Pantoja 
- Evelyn Alzier 
- Vitor Santis 
- Antônio Fascio

**Pull request merged**